index.html

```diff
@@ -97,7 +97,7 @@ <h3>Multi-Post Stories Gain + Glory</h3>
                             <li>html</li>
                         </ul>
     
-                        <a href="#" class="action-btn">See Project</a>
+                        <a href="#" class="action-btn" data-id="1">See Project</a>
                     </div>
                 </div>
                 <div class="works-grid-card">
```

Comment:
- [ ] It is required that the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed, and instead it should be added via JavaScript. This is contained in the project requirements. Kindly take a look at the project requirements once again, remove already created html codes for the project sections and create them dynamically from your js file.

![project-rqts](https://user-images.githubusercontent.com/26597432/215899479-bd7bcf80-7182-4a86-9346-149b78a30d24.png)